frontend/src/metabase/admin/datamodel/components/database/MetadataSchemaList.jsx

```diff
@@ -49,7 +48,9 @@ export default class MetadataSchemaList extends Component {
         </div>
         <ul className="AdminList-items">
           <li className="AdminList-section">
-            {filteredSchemas.length} {inflect("schema", filteredSchemas.length)}
+            {(n => ngettext(msgid`${n} schema`, `${n} schemas`, n))(
```

Comment:
Would it be worth trying to create a function to help make this a little more accessible or would be be reinventing the wheel? It feels a little low level to have to import `ngettext, msgid` and remember this particular syntax every time we need to get the right inflection.

Reply:
Unfortunately I think it's necessary to use these specific functions so `c-3po` / `ttag` (apparently it was renamed) can work correctly: https://c-3po.js.org/why-use-msgid-for-ngettext.html

The `(n => ...)(...)` syntax is just me being a little (too?) clever to make it less verbose, otherwise a potentially long expression would be repeated 3 times. I'm open to other suggestions, or we can just deal with it being verbose.

Reply:
Gotcha. This is probably fine for now then 👍